js/views/TestProfilePage.js

```diff
@@ -45,8 +69,15 @@ export default class extends BaseVw {
     social.remove(social.at(index));
   }
 
+  onRemoveSocial() {
+    // since the validation is index based and
+    // removing alters the indexes in the collection,
+    // we need to revalidate
```

Comment:
We could assign the items ids of some kind instead. We can revisit it if it becomes an issue later.